### ISSUE TITLE
LanguagePrimitives.EnumToValue is (no longer) resolved

### ISSUE THREAD

**Issue by 0x53A** [fable-compiler/Fable#1813]
searching through github, it seems this was reported with https://github.com/fable-compiler/Fable/issues/1521 and fixed with https://github.com/fable-compiler/Fable/pull/1522.

But in the current REPL, this fails:

```f#
// CLR enum
type TE = A = 1 | B = 2

let x = TE.A

let ix = LanguagePrimitives.EnumToValue x
let x2 : TE = LanguagePrimitives.EnumOfValue ix


```

![image](https://user-images.githubusercontent.com/4236651/56056144-ab5e4580-5d5b-11e9-86e2-222dac5fa7e0.png)

[repl link](https://fable.io/repl/#?code=LYewJgrgNgpgBABQE4gOZIIbALACg8D0BcAwgDIBKcMAdhDrgC4CeADvACoCicAvHAEE+cAIxwAPnABCwgEx48sRnAAew7gDoBC3ErgBLNfzIYaqCBlQxk+4Psb6AbjADOGrnWAcQANQxQIeBVFGGUVWTgALjhuYRMzCysbOwdnNw96AHkAMz8A+EMdIA&html=DwCwLgtgNgfAsAKEaApgQwCbwQAjz4CFMNHcMABwFoUBHAVwEsA3AXgHIBhAewDswU-KgBUAnhRTscAYz4D+HAQA8wAenDQA3DJBoATgGdiremABmVABzts+Agel7GFMDgN7prAEQB9HwAkAeQBlYT9VKEYAIwNVPXRpMAA6Cj1uDHpExj4kiEZeJIArAy8YYFUHJxdbfGBK51d3T18AkLCfCOjY+LREqgxuCBS0jKycvILi0vL66uR1dCxEZCj00RkoNAMDbzQKall+NHyUPVLEO2AMFhxGDG8UKDyDECo9ii8NrZ2vR+fX97TVTXZjYcqrDCibDzDSwRBAA&css=BYFwtgNgNAsAUAIwPYBMCeACA3vDeMBmSAdiALQDOAlgF4CmAXBgIwBsADgB4Dc8AvvHgA6MAEMqxMgGMSIccToAnbLnwoqFdhFFomBCHR6q8AdyooQwJswAMNgKS84+DMDpUA5qGt3HxjABWAK4UIFQEaNKydKRMUjEgSk4uohCeklSJYBRxCUn++oZk6op0UmEkcUgQQWDETgJwwhLsQSAqzvjsoijqxB5MQgBMAKylYMn4RKSUtIwsHEadpuaWTKM2XJN4YooeEmTIICBIYNbjDfBAA)

**Comment by ncave** [comment]
#1522 doesn't actually address this, but #1814 will.
